src/filters/fs.rs

```diff
@@ -132,12 +132,14 @@ fn sanitize_path(base: impl AsRef<Path>, tail: &str) -> Result<PathBuf, Rejectio
     Ok(buf)
 }
 
-#[derive(Debug)]
-struct Conditionals {
-    if_modified_since: Option<IfModifiedSince>,
-    if_unmodified_since: Option<IfUnmodifiedSince>,
-    if_range: Option<IfRange>,
-    range: Option<Range>,
+/// Conditionals that define certain aspects of file serving. These are usually based on the headers of a request, though they can be manually constructed.
+#[allow(missing_docs)] // TODO add docs for these (consult @seanmonstar)
+#[derive(Debug, Default)] // We derive `Default` as well to make constructing this manually easier
```

Comment:
I think you can drop the comment here, since the whole reason of having a `Default` impl is kind of captured by `Default`'s own doc comments anyways.